routes/api/services/read-services-category.js

```diff
@@ -7,5 +7,24 @@ var router = express.Router();
 
 console.log('read-services-category');
 
+
+app.get("/api/volunteers", function(req, res) {
+    // 1. Add a join to include all of users
+    db.User.findAll({}).then(function(dbVolunteer) {
+      res.json(dbVolunteer);
+    });
+  });
+
+  app.get("/api/volunteers/:id", function(req, res) {
+    // 2; Add a join to include all of users
+    db.Author.findOne({
+      where: {
+        id: req.params.id
+      }
+    }).then(function(dbVolunteer) {
+      res.json(dbVolunteer);
+    });
+  });
+
```

Comment:
Let's remove these and replace them with the service category routes we stuck in `read-services.js`

If you get a chance, I'm thinking we should rename this file to `read-service-categories.js`